src/plasma/plasma_manager.c

```diff
@@ -1515,6 +1520,16 @@ void process_object_notification(event_loop *loop,
   entry->object_id = obj_id;
   HASH_ADD(hh, state->local_available_objects, object_id, sizeof(object_id),
            entry);
+
+  /* Add this object to the (redis) object table. */
+  if (state->db) {
+    LOG_DEBUG("Publishing to object table from DB client %d.",
+      get_client_id(conn->manager_state->db));
+      /* TODO(swang): Log the error if we fail to add the object, and possibly
```

Comment:
Actually, can you get rid of the `LOG_DEBUG` statement? It looks like we had a bug there because `get_client_id` doesn't return an int, so that statement won't work.